pkg/roachprod/k8s/roachprod-gc.yaml

```diff
@@ -10,7 +10,7 @@
 #    --from-literal azure_password=XYZZY
 #    --from-literal azure_tenant_id=XYZZY
 #    --from-literal slack_token=XYZZY
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
```

Comment:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/

>The batch/v1beta1 API version of CronJob is no longer served as of v1.25.

>Migrate manifests and API clients to use the batch/v1 API version, available since v1.21.